bin/improver-continuous-snow-falling-level

```diff
@@ -0,0 +1,98 @@
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+# ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+# LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+# CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+# SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+# INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+# CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+# ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+# POSSIBILITY OF SUCH DAMAGE.
+"""Script to apply thresholding to a cube."""
+
+import argparse
+import iris
+
+from improver.psychrometric_calculations.psychrometric_calculations import (
+    FallingSnowLevel)
+
+
+def main():
+    """Load in arguments and get going."""
+    parser = argparse.ArgumentParser(
+        description="Calculate the continuous falling snow level ")
+    parser.add_argument("temperature", metavar="TEMPERATURE",
+                        help="File path to a cube of air temperatures at"
+                        " heights (m) at the points for which the continuous "
+                        "falling snow level is being calculated.")
+    parser.add_argument("relative_humidity", metavar="RELATIVE_HUMIDITY",
+                        help="File path to a cube of relative_humidities at"
+                        " heights (m) at the points for which the continuous "
+                        "falling snow level is being calculated.")
+    parser.add_argument("pressure", metavar="PRESSURE",
+                        help="File path to a cube of air pressures at"
+                        " heights (m) at the points for which the continuous "
+                        "falling snow level is being calculated.")
+    parser.add_argument("orography", metavar="OROGRAPHY",
+                        help="A path to an input NetCDF file containing "
+                        "a cube with the orography in m of the terrain "
+                        "that the continuous falling snow level is being"
+                        "calculated.")
+    parser.add_argument("output_filepath", metavar="OUTPUT_FILE",
+                        help="The output path for the processed NetCDF")
+    parser.add_argument("--precision", metavar="NEWTON_PRECISION",
+                        default=0.005, type=float,
+                        help="Precision to which the wet bulb temperature "
+                        "is required: This is used by the Newton iteration "
+                        "default value is 0.005")
+    parser.add_argument("--falling_level_threshold",
+                        metavar="FALLING_LEVEL_THRESHOLD",
+                        default=90.0, type=float,
+                        help="Cutoff threshold for the wet-bulb integral used "
+                        "to calculate the falling snow level.  "
```

Comment:
`Cutoff threshold for the wet-bulb integral used to calculate the falling snow level. This threshold indicates the level at which falling snow is deemed to have melted to become rain. The default value is 90.0, an empirically derived value.`

Reply:
Done